bin/ksh/config.h

```diff
@@ -229,7 +229,7 @@
 #define POSIXLY_CORRECT 1
 
 /* Specify default $ENV? */
-#define DEFAULT_ENV	"$HOME/.kshrc"
+#define DEFAULT_ENV	"/etc/.kshrc"
```

Comment:
Please mark `DEFAULT_ENV` with comment that we should restore it to original state when user home directories are available for write.